spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaAutoConfiguration.java

```diff
@@ -157,6 +159,8 @@ public Server armeriaServer(
             }
             return result;
         }).join();
+
+        eventPublisher.publishEvent(new ArmeriaServerStartedEvent(server));
```

Comment:
Probably better to use [`ServerListener`]( `https://line.github.io/armeria/apidocs/com/linecorp/armeria/server/ServerListener.html) to listen to the server started event?

Additionally, one of our internal users asked us about how to listen to the server life cycle event in Spring. Maybe this PR is very useful for that case.
Maybe, in the following PR, we can add other events such as `ArmeriaServerStartingEvent`, `ArmeriaServerStopingEvent`...?

Reply:
I'll check. :wink:

Reply:
I found that publishing events from other threads doesn't work as expected. This is [a problem with Spring itself](https://github.com/spring-projects/spring-framework/issues/20904) and it seems to be impossible to use `ServerListener` for now.